.circleci/config.yml

```diff
@@ -1,52 +1,65 @@
-# Python CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-python/ for more details
-#
 version: 2
 jobs:
-  build:
```

Comment:
If we're going to "rebuild" for every test, it'd be faster to add 1 build step and use docker save to save the artifact and docker load on each of the test phases.